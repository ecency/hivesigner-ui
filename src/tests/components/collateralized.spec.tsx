import React from "react";
import From from "../../components/Sign/items/collateralized";
import renderer from "react-test-renderer";
describe("collateralized component Test", () => {
  test("UI Test", () => {
    const FromRender = renderer.create(<From />);
    let tree = FromRender.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
