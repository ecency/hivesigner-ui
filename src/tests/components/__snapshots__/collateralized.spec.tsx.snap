// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`collateralized component Test UI Test 1`] = `
<div
  className="Item"
>
  <div
    className="ItemHeader"
    id="transfer"
    onClick={[Function]}
  >
    <svg
      className="Item_icon_rotate"
      fill="currentColor"
      height="1em"
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 1024 1024"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
      />
    </svg>
    <div
      className="Item_title"
    >
      Collateralized Convert
    </div>
  </div>
</div>
`;
